src/common/src/RnNoiseCommonPlugin.cpp

```diff
@@ -16,7 +17,9 @@ void RnNoiseCommonPlugin::deinit() {
     m_denoiseState.reset();
 }
 
-void RnNoiseCommonPlugin::process(const float *in, float *out, int32_t sampleFrames) {
+void RnNoiseCommonPlugin::process(const float *in, float *out, int32_t sampleFrames, float vadThreshold) {
+    assert(vadThreshold >= 0.f && vadThreshold <= 100.f);
```

Comment:
Oh, did notice one thing.

Shouldn't this check if `vadThreshold <= 1.f`? I think the process function gets passed

    vad_threshold_normalized = std::max(std::min(vad_threshold / 100.f, 99.f), 0.f);

Which is the value a float between 0 and 1 instead of the percentage?

Reply:
Yes, you are right.